vng_api_common/audittrails/viewsets.py

```diff
@@ -55,6 +55,8 @@ def create_audittrail(self, status_code, action, version_before_edit, version_af
             applicatie_weergave=applicatie_weergave,
             actie=action,
             actie_weergave=CommonResourceAction.labels.get(action, ''),
+            gebruikers_id=self.request.jwt_auth.payload['user_id'],
```

Comment:
This doesn't have a fallback to the NLX header - if it's not present in the JWT, you should look up the NLX header, and if that isn't available, save it as an empty string probably.